### ISSUE TITLE
Asset file has wrong casing, causing 404 on case-sensitive OS

### ISSUE THREAD

**Issue by brentheimer** [dawoe/umbraco-multilanguage-textbox#4]
Asset file has wrong casing, causing 404 on linux hosted environment.

In the package manifest `controller.js` comes from `/App_Plugins/Our.Umbraco.MultilanguageTextbox/` but the actual folder is `/App_Plugins/Our.Umbraco.MultiLanguageTextbox/`

See line by line for comparison:
```
/App_Plugins/Our.Umbraco.MultiLanguageTextbox/editor.html
/App_Plugins/Our.Umbraco.MultilanguageTextbox/controller.js
```
Updating the `L` in the path should resolve the issue.

**Comment by brentheimer** [comment]
PR #5 fixes this.

**Comment by dawoe** [comment]
Hi @brentheimer 

This is fixed in 10.1.0 which will be on nuget soon
